conf/modules.config

```diff
@@ -10,8 +10,29 @@
 ----------------------------------------------------------------------------------------
 */
 
-// TODO tie all ext.args to nextflow.config
-// TODO mix in versions of all outputs
+
+def reformat_output(filename_in, outputtype, merge_string, meta_){
```

Comment:
Do you know if there's a way to move this into a Groovy file and import into the config file rather than having the function defined in the config file.

Ideally, I'd like there to be unit tests for this, but nf-test seems to only let you test functions in Nextflow scripts or Groovy file: https://www.nf-test.com/docs/testcases/nextflow_function/

Reply:
I do not know currently. I did try defining the function in other places but there was always issues. I beleive when nextflow compiles the configs that it just inlines everything in the `nextflow.config` I have tried creating function in groovy files previously but I was never able to get them to work.

I dont know if it is any consilation and I know it is not a unit test, but the function is tested indirectly as the output paths and names are updated and tests all passed.

Reply:
Thanks. I don't know if any way to have this separated into a Groovy file either. I think having this function in the config here is okay. And I am glad it is tested.